packages/docs/src/pages/getting-started/kitchensink.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div>
+    <docs-content :config="config" />
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { config } from '@/page-configs/getting-started/kitchensink/page-config'
```

Comment:
Needs relative paths :X.

Reply:
Maybe we can use absolute path here? Here I need to go upstairs to `src` folder and then downstairs. Why can't we just start immediatly from `@ === src` ?